kernel_gateway/tests/services/cell/test_parser.py

```diff
@@ -5,7 +5,7 @@
 import unittest
 import re
 import sys
-from kernel_gateway.services.cell.parser import APICellParser
```

Comment:
We should refactor the test folders to following the renaming of the services folders? For example, this one should be in `tests/notebook_http/cell/test_parser.py` probably.